device-types/QNAP/TS-h1277AXU-RP.yml

```diff
@@ -4,20 +4,12 @@ model: TS-h1277AXU-RP
 slug: qnap-ts-h1277axu-rp
 u_height: 2
 is_full_depth: true
+airflow: front-to-rear
 weight: 19
 weight_unit: kg
 comments: '[QNAP TS-h1277AXU-RP](https://www.qnap.com/en/product/ts-h1277axu-rp)'
 front_image: true
 rear_image: true
-power-ports:
-  - name: PSU 1
-    type: iec-60320-c14
-    maximum_draw: 550
-    allocated_draw: 370
-  - name: PSU 2
-    type: iec-60320-c14
-    maximum_draw: 550
-    allocated_draw: 370
```

Comment:
It doesn't look like PSU are modulable, as both are available by default: [*Power Supply Unit 	550W(x2), 100-240V*](https://www.qnap.com/en/product/ts-h1277axu-rp/specs/hardware)

Reply:
As the PSUs are Field replaceable units they should be modeled as module-bays